workers/unity/Packages/com.improbable.gdk.mobile/Editor/MobileLaunchUtils.cs

```diff
@@ -0,0 +1,44 @@
+using System.Text;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Tools;
+using UnityEngine;
+
+namespace Improbable.Gdk.Mobile
+{
+    public class MobileLaunchUtils
+    {
+        private const string argStructure = "+{0} {1} ";
+
+        public static bool TryPrepareArguments(bool shouldConnectLocally, string runtimeIp, out string builtArguments)
```

Comment:
Why not throw instead of returning a bool and having an out parameter? Afaict if there's an error its a "stop the world" scenario.